content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Enabling CSM Cloud Workload Security on Kubernetes
+kind: documentation
+code_lang: kubernetes
+type: multi-code-lang
+code_lang_weight: 60 # a number that represents relative weight. 
+---
+
+Use the following instructions to enable [CSM Threats][1] on Kubernetes. To learn more about the supported deployment types for each CSM feature, see [Setting Up Cloud Security Management][2].
+
+{{< tabs >}}
+
+{{% tab "Operator" %}}
+
+1. Add the following to the `spec` section of the `values.yaml` file:
+
+    ```yaml
+    # values.yaml file
+    spec:
+      features:
+        remoteConfiguration:
+          enabled: true
+        cws:
+          enabled: true
+    ```
+
+2. Restart the Agent.
+
+
+[2]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.v2alpha1.md
+
+{{% /tab %}}
+
+{{% tab "Helm" %}}
+
+1. Add the following to the `datadog` section of the `values.yaml` file:
```

Comment:
```suggestion
1. Add the following to the `datadog` section of the `datadog-values.yaml` file:
```

container docs generally refer to the custom helm config file as `datadog-values.yaml`—the `values.yaml` file is the default helm chart config, where parameters not set in `datadog-values.yaml` default to those in `values.yaml`

Reply:
Didn't realize that. That's good to know! Thanks for the heads up.